codalab/model/bundle_model.py

```diff
@@ -1831,6 +1831,60 @@ def add_user(self, username, email, first_name, last_name, password,
 
         return user_id, verification_key
 
+    def delete_user(self, user_id=None):
+        with self.engine.begin() as connection:
+
+            # ==User verification==
```

Comment:
My preference would be to make each `execute` command one line and remove all the newlines between statements in the same logical grouping...  In general, putting everything on one line is a bad idea and hard to read, but in this case, I think it makes it easier to see the overall structure, which is harder to get right than the individual details.  (I'm really worried about adding a table somewhere and forgetting to update this.)